docs/src/filters/writing_custom_filters.md

```diff
@@ -3,27 +3,27 @@
 Quilkin provides an extensible implementation of [Filters] that allows us to plug in custom implementations to fit our needs.
 This document provides an overview of the API and how we can go about writing our own [Filters].
 
-#### API Components
+## API Components
```

Comment:
Just noting: Since I had to remove the lists, I moved the headers to be in level order. PTAL, let me know what you think.